gtdynamics/universal_robot/Link.h

```diff
@@ -121,14 +117,8 @@ class Link : public boost::enable_shared_from_this<Link> {
   /// add joint to the link
   void addJoint(const JointSharedPtr &joint) { joints_.push_back(joint); }
 
-  /// transform from link to world frame
-  const gtsam::Pose3 &wTl() const { return wTl_; }
-
-  /// transform from link CoM frame to link frame
-  const gtsam::Pose3 &lTcom() const { return lTcom_; }
-
-  /// transform from link CoM frame to world frame
-  inline const gtsam::Pose3 wTcom() const { return wTl() * lTcom(); }
+  /// transform from base frame to link COM frame
+  inline const gtsam::Pose3 bTcom() const { return bMcom_; }
```

Comment:
I think it should be `bMcom`. We should use `M` any time we are talking about a pose at rest, I.e., as specified in SDF.

Reply:
done